packages/wrangler/src/utils/retry.ts

```diff
@@ -0,0 +1,18 @@
+import { setTimeout } from "node:timers/promises";
+
+export async function retryOnError<T>(
+	action: () => T | Promise<T>,
+	backoff = 2_000,
```

Comment:
this signature means you have to pass a `backoff` if you want to define `attempts`, which isn't ideal. Can we put backoff and attempts into an `options` object instead, with both fields being optional?